src/validation.cpp

```diff
@@ -4272,8 +4272,9 @@ bool LoadMempool(void)
     }
 
     int64_t count = 0;
-    int64_t skipped = 0;
+    int64_t expired = 0;
     int64_t failed = 0;
+    int64_t alreadythere = 0;
```

Comment:
nit: snake case `already_there`

Reply:
IMO consider renaming to one of `already_imported`, `already_loaded`, `already_in_mempool`.

Reply:
Personally feel the current name is fine as is, but will change if people want it.